### PR TITLE
Add commandline documentation to the website

### LINKED ISSUE #137: Add command line documentation to website

### PR DESCRIPTION
**Pull request by juliaogris**
Fixes #137.

Changes proposed in this pull request:
- Add several simple standalone sysl example files with usage instruction
- Add command line documentation to sysl.io website
- Fix various typos

@anz-bank/sysl-developers

### CODE REVIEW

File: demo/simple/reljam-view.sysl [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# reljam view /reljam-View GroupModelToApi
+# TODO
+
+GoupModel [package="io.sysl.demo.view.model"]:
```
Comment (sahejsingh) [inline demo/simple/reljam-view.sysl]: GoupModel?

File: demo/simple/sysl-ints.sysl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# sysl ints -o "sysl-ints.png" /sysl-ints --project Project -v
+# sysl ints -o "sysl-ints-epa.png" /sysl-ints --project Project --epa -v
+IntegratedSystem:
+    _1:
```
Comment (sahejsingh) [inline demo/simple/sysl-ints.sysl]: _1 is confusing. IMHO might as well use this to give name to your intent.

File: docs/website/content/docs/quickstart.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ draft: false
 bref: "Create your first diagram from a Sysl specification"
 toc: false
 ---
+
+Sysl consists of two executables: `sysl` and `reljam`.</br> The **Sy**stem **S**pecification **L**anguage `sysl` is mainly concerned with diagram creation whereas the **Rel**ational **Ja**va **M**odel program `reljam` generates different types of source code output.
+
 Install Sysl
 ------------
 Windows users can run standalone Sysl executables whereas users of other operating systems need to work with either Python or Docker.
```
Comment (sahejsingh) [inline docs/website/content/docs/quickstart.md]: Section on Docker needs to be enhanced.
1. Add -v command to mount the volume
2. Add section on to pull plantuml docker image, run the image on 8080. also change SYSL_PLANTUML to this local image.

File: demo/simple/reljam-view.sysl (hunk 1 above)
Comment (juliaogris) [reply demo/simple/reljam-view.sysl]: Yeah, not idea, I agree. Maybe I will just remove the whole file.
For context: I wanted to create a simplified Petshop sample. So I thought I model a class in a school with students. Then I realised that the word class is quite overloaded in IT, so I changed it to group.
I will remove this.

File: demo/simple/sysl-ints.sysl (hunk 2 above)
Comment (juliaogris) [reply demo/simple/sysl-ints.sysl]: Well, the intent is to create a minimal sysl file for an integration diagram.

I used `_1` purposefully to demonstrate that it doesn't matter what you put here - it doesn't show up in the output. Do you have any other naming suggestions that make this point clear and are meaningful at the same time? Maybe something like `INTEGRATED_ENDPOINT_1` or `hidden_integrated_endpoint1` ?

Ideally IMO we wouldn't need `_1` or similar at all, but that option doesn't exist.

File: docs/website/content/docs/quickstart.md (hunk 3 above)
Comment (juliaogris) [reply docs/website/content/docs/quickstart.md]: hmm, you have got issue #121  (Document docker usage) assigned to you. once you have done it for the Readme.rst I'm happy to transfer it to the website (which is really what I've done here too).

File: demo/simple/sysl-ints.sysl (hunk 2 above)
Comment (sahejsingh) [reply demo/simple/sysl-ints.sysl]: Fair enough.

Comment (juliaogris) [reply demo/simple/sysl-ints.sysl]: I have actually changed it now to `integrated_endpoint_1` ...

**Pull request merged**